gw_eccentricity/eccDefinition.py

```diff
@@ -981,11 +1054,94 @@ def get_default_extrema_finding_kwargs(self, width):
             "rel_height": 0.5,
             "plateau_size": None}
         return default_extrema_finding_kwargs
+    
+    def get_default_special_interp_kwargs_for_extrema(self):
+        """Get default kwargs to be passed to extrema interpolating method.
+        
+        Accurate interpolation of omega_gw extrema is crucial for obtaining
+        monotonic eccentricity evoultion with time. Depending on the
+        interpolation method, this function returns the default kwargs to be
+        passed to the interpolating function. 
+        """
+        allowed_methods = self.extra_kwargs["special_interp_kwargs_for_extrema"].keys()
```

Comment:
Confused. Why are allowed_methods in self.extra_kwargs["special_interp_kwargs_for_extrema"]?

Reply:
Fixed